repos/keg-cli/src/utils/log/logVirtualUrl.js

```diff
@@ -5,6 +5,8 @@ const { TAP_URL } = require('KegConst/constants')
  * Helper that Gets and logs the Virtual Machines IP address
  */
 const logVirtualUrl = (context) => {
+  if(context === 'proxy') return Logger.empty()
+
   const useUrl = context
```

Comment:
we should also update this url now so it's displaying the correct url
![image](https://user-images.githubusercontent.com/3317835/98046885-38f30180-1de8-11eb-9a85-56385cec8b3f.png)

Reply:
Ahh good point. Ill update that

Reply:
Ok, this has been updated. Also added a few more tests for the logVirtualUrl helper
  * Had to update the buildComposeCommand a bit, to get access to the generated url
  * Also added it to the `docker package run` task, which is helpful; cause who knows what branch the image label has

Reply:
noice